src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -149,14 +153,28 @@ public static void setVideoRecorderToNull() {
         videoRecorder = null;
     }
 
+    /**
+     * Checks if video recording enabled
+     *
+     * @return true if video.enabled property defined as true
+     */
+    public static boolean videoRecorderIsEnabled() {
+        if (!videoRecorderStatusChecked) {
+            String video = Props.get("video.enabled", "false");
+            videoRecorderStatusChecked = true;
+            videoRecorderEnabled = !(null == video || video.equalsIgnoreCase("false") || video.isEmpty());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "false" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)